### PR TITLE
Support setting exec_mem_limit in ExportJob

### PR DESCRIPTION
**Pull request by chaoyli**

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/load/ExportJob.java [hunk 1]
```diff
@@ -595,6 +605,10 @@ public void readFields(DataInput in) throws IOException {
         columnSeparator = Text.readString(in);
         lineDelimiter = Text.readString(in);
 
+        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_52) {
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/load/ExportJob.java]:
```suggestion
        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_53) {
```

File: fe/src/main/java/org/apache/doris/analysis/ExportStmt.java [hunk 2]
```diff
@@ -221,10 +229,17 @@ private static void checkPath(String path) throws AnalysisException {
...
         this.columnSeparator = PropertyAnalyzer.analyzeColumnSeparator(
                 properties, ExportStmt.DEFAULT_COLUMN_SEPARATOR);
         this.lineDelimiter = PropertyAnalyzer.analyzeLineDelimiter(properties, ExportStmt.DEFAULT_LINE_DELIMITER);
+        if (properties != null && properties.containsKey(LoadStmt.EXEC_MEM_LIMIT)) {
+            try {
+                this.execMemLimit = Long.parseLong(properties.get(LoadStmt.EXEC_MEM_LIMIT));
+            } catch (NumberFormatException e) {
+                throw new DdlException("Execute memory limit is not Long", e);
+            }
+        }
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/analysis/ExportStmt.java]: if properties not contain this property, you can get it from SessionVariable.

**Review by imay — CHANGES_REQUESTED** [review]

File: fe/src/main/java/org/apache/doris/analysis/ExportStmt.java (hunk 2 above)
Comment (chaoyli) [reply fe/src/main/java/org/apache/doris/analysis/ExportStmt.java]: OK

File: fe/src/main/java/org/apache/doris/load/ExportJob.java (hunk 1 above)
Comment (chaoyli) [reply fe/src/main/java/org/apache/doris/load/ExportJob.java]: OK

**Review by imay — APPROVED** [review]

**Pull request merged**